docs/DeveloperGuide.adoc

```diff
@@ -844,6 +846,179 @@ The following activity diagram below shows how the `alias` command works.
 .Activity Diagram for sample alias command.
 image::AliasActivityDiagram.png[width=650, align="center"]
 
+//tag::info[]
+=== Info Feature
+
+The info feature allows users to view more information about a particular command. This can be done through calling the
+`info` command.
+
+==== Classes for info feature in Model
+
+In order to support the `info` command, new classes were added that correspond to each command in the app.
+This is to allow for easy access and also to better organize the information to be displayed for each commands.
+
+These classes only has public fields without any methods as it is only needed by `InfoWindow` to access the information
+stored within the class.
+
+The classes added where each class represents an info card:
+
+1. `AddInfo`
+2. `AliasInfo`
+3. `AutoSortInfo`
+4. `BudgetInfo`
+5. `BuyInfo`
+6. `ClearInfo`
+7. `CustomSortInfo`
+8. `DefaultInfo`
+9. `DeleteInfo`
+10. `DislikeInfo`
+11. `EditInfo`
+12. `ExitInfo`
+13. `FilterInfo`
+14. `HelpInfo`
+15. `HistoryInfo`
+16. `InfoInfo`
+17. `LikeInfo`
+18. `MakeSortInfo`
+19. `RecommendInfo`
+20. `RemoveDislikeInfo`
+21. `RemoveLikeInfo`
+22. `SaveInfo`
+23. `ShowInfo`
+24. `ThemeInfo`
+25. `TopUpInfo`
+26. `ViewSortInfo`
+27. `WithdrawInfo`
```

Comment:
Should not use numbering. Inconsistent with current DG formatting. Just use "====="